cloudbridge/providers/azure/azure_client.py

```diff
@@ -181,13 +180,13 @@ def __init__(self, config):
         log.debug("azure subscription : %s", self.subscription_id)
 
     @property
-    @tenacity.retry(stop=tenacity.stop_after_attempt(5), reraise=True)
+    @tenacity.retry(stop=tenacity.stop.stop_after_attempt(5), reraise=True)
```

Comment:
pylance was complaining and this seems to make it happy :shrug:  I'm happy to revert the change

Reply:
May have been a change in the latest version of tenacity? Fine if it's happy.